### PR TITLE
:sparkles: Add --no-color flag

### PR DESCRIPTION
**Pull request by genevieveluyt**
##### Description

Adds `--no-color` flag which can be used to disable printing color codes in the output. Colored output is intended for stdout but displays poorly elsewhere.

For example, redirecting the default "pretty" output to a file:

```
[36m
---------------- Results for ---------------

[0m[36m  apiVersion: v1
[0m[36m  kind: Namespace
[0m[36m  metadata:
[0m[36m    name: deployment-apps-v1
[0m[36m
--------------------------------------------
```

##### Type of change

- [ ] Bug fix :bug:
- [x] New feature :sparkles:
- [x] This change requires a documentation update :book:
- [ ] Breaking changes :warning:
##### How Has This Been Tested?

- [ ] Ran `go run cmd/main.go all -f "internal/test/fixtures/all_resources/deployment-apps-v1.yml" > out.txt` and compared it with `go run cmd/main.go all --no-color -f "internal/test/fixtures/all_resources/deployment-apps-v1.yml" > out.txt`

##### Checklist:

- [x] I have :tophat: my changes (A 🎩 specifically includes pulling down changes, setting them up, and manually testing the changed features and potential side effects to make sure nothing is broken)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] The test coverage did not decrease
- [ ] I have signed the appropriate [Contributor License Agreement](https://cla.shopify.com/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,13 +167,13 @@ For more information on kubernetes config files, see https://kubernetes.io/docs/
 
 Kubeaudit produces results with three levels of severity:
 
-`Error`: A security issue or invalid kubernetes configuration
-`Warning`: A best practice recommendation
-`Info`: Informational, no action required. This includes results that are [overridden](#override-errors)
+- `Error`: A security issue or invalid kubernetes configuration
+- `Warning`: A best practice recommendation
+- `Info`: Informational, no action required. This includes results that are [overridden](#override-errors)
```
Comment (genevieveluyt) [inline README.md:172]: This is an unrelated change, just improves the formatting by separating them severity codes by newlines

**Review by cailynse — APPROVED** [review]

**Review by Owen-Cummings — APPROVED** [review]
As a Canadian: Color vs Colour 👀  But otherwise :shipit:

**Comment by genevieveluyt** [comment]
The failing CLA check is due to [a bug](https://github.com/Shopify/cla/issues/609). Since there's no ETA to fix, we've got the ok to merge through the failing check (Shopify employees don't have to sign the CLA, they should always pass the check).
